### ISSUE TITLE
Connected to WIFI, but no devices found in Amazon Alexa app

### ISSUE THREAD

**Issue by terrag42** [thehookup/Alexa-Enabled-LED-Clock#1]
I connected to the "clock setup" SSID and entered my wifi credentials using wifi manager. I also set invocation words like this:
Grid
Grid two
Grid three
Grid Timer

For timezone, I entered "-4"
Then the ESP8266 successfully connected to my wifi, and I launched my Alexa app. 
I added a device, selected "Light", then under "what brand?" I selected "other". then "discover devices". No devices were discovered. Do I need to enable the Philips Hue skill for this to work? Thanks.

**Comment by terrag42** [comment]
To help with troubleshooting, I am pasting the device information obtained by pointing my browser to the ip address of my Wemos D1 Mini, like this: 
192.168.x.xxx/espalexa

Hello from Espalexa!

Value of device 1 (): 100 (Color light, colormode=xy, r=255, g=0, b=0, ct=500, hue=0, sat=0, x=0.70, y=0.30)
Value of device 2 (): 13 (Color light, colormode=xy, r=0, g=255, b=0, ct=500, hue=0, sat=0, x=0.17, y=0.75)
Value of device 3 (): 127 (Color light, colormode=xy, r=0, g=0, b=255, ct=500, hue=0, sat=0, x=0.14, y=0.04)
Value of device 4 (): 0 (Color light, colormode=xy, r=254, g=221, b=80, ct=500, hue=0, sat=0, x=0.50, y=0.50)

Free Heap: 34664
Uptime: 1186666

Espalexa library v2.4.2 by Christian Schwinne 2019

**Comment by terrag42** [comment]
More info: I compiled with the latest Arduino IDE (version 1.8.9), and I'm running the latest Arduino core (verson 2.5.0) and I tried unplugging and re-powering my genuine Amazon alexa dot, still no luck.
Thanks!

**Comment by terrag42** [comment]
The output from the Espalexa devices shows that there's nothing between my parentheses(). I suspected that they were not being set up correctly, and I "hard-coded" some values without relying on wifimanager to put the strings in there. This worked, so I suspect that I have one of my library with the same names that's not the same one that this sketch uses. Since I got it working, I'm closing the issue, but still haven't pinpointed which library it is (maybe the .json library?)
